src/pages/demo/SendPageErrors.vue

```diff
@@ -192,7 +192,11 @@ export default defineComponent({
                 console.error(err);
                 if (err instanceof AntelopeError) {
                     const evmErr = err as AntelopeError;
-                    ant.config.notifyFailureMessage(this.$t(evmErr.message), evmErr.payload);
+                    if (evmErr.message === 'antelope.evm.error_transaction_canceled') {
```

Comment:
i think it would be good to make this into a utility function because it is used the same in several places, and will be copied for staking too, something like 

```ts
export function handleTransactionError(
    err: AntelopeError,
    ant: Antelope,
) {
    if (err.message === 'antelope.evm.error_transaction_canceled') {
        ant.config.notifyNeutralMessageHandler(this.$t(err.message));
    } else {
        ant.config.notifyFailureMessage(this.$t(err.message), err.payload);
    }
}


// in components
handleTransactionError(evmErr, ant);
```

Reply:
awesome idea !!! :muscle: